string goal_id
float32 goal_limit
float32 wait_time

float32 x
float32 y
float32 z
float32 yaw

float32 x_vel
float32 y_vel
float32 z_vel
float32 yaw_vel

float32 x_acc
float32 y_acc
float32 z_acc
float32 yaw_acc
